subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/results/PerformanceTestResult.java

```diff
@@ -37,13 +37,8 @@ public  PerformanceTestResult() {
         whereAmI = new Throwable();
     }
 
-    protected static Checks whatToCheck() {
-        Checks result = Checks.ALL;
-        String override = System.getProperty("org.gradle.performance.execution.checks");
-        if (override != null) {
-            result = Checks.valueOf(override.toUpperCase());
-        }
-        return result;
+    public static boolean hasChecks() {
```

Comment:
It's good that this is now simplified since we do not use the different check types.

Can you add a comment here what `!hasChecks()` means and for which scenario that is useful? Because it almost sounds like the test would check nothing.